core/templates/components/state-editor/state-editor.component.ts

```diff
@@ -76,7 +76,7 @@ export class StateEditorComponent implements OnInit, OnDestroy {
   @Input() interactionIsShown!: boolean;
   @Input() stateContentSaveButtonPlaceholder!: string;
   @Input() stateContentPlaceholder!: string;
-
+  @Input() isSticky: boolean = false;
```

Comment:
Why is this `false` by default?

Reply:
@rassa328 Can you instead make this a required parameter and specify it everywhere this component is called?